app/scripts/app.js

```diff
@@ -0,0 +1,55 @@
+/* global $ */
+'use strict';
+
+/**
+ * @ngdoc overview
+ * @name flightNodedemoApp
+ * @description
+ * # flightNodedemoApp
+ *
+ * Main module of the application.
+ */
+angular
+  .module('flightNodedemoApp', [
+    'ngAnimate',
+    'ngCookies',
+    'ngResource',
+    'ngRoute',
+    'ngSanitize',
+    'ngTouch'
+  ])
+  .config(function ($routeProvider) {
+    $routeProvider
+      .when('/', {
+        templateUrl: 'views/main.html',
+        controller: 'MainCtrl'
+      })
+      .when('/about', {
+        templateUrl: 'views/about.html',
+        controller: 'AboutCtrl'
+      })
+      .when('/login', {
+        templateUrl: 'views/login.html',
+        controller: 'LoginController'
+      })
+      .otherwise({
+        redirectTo: '/'
+      });
+  })
+  .directive('loading', function () {
```

Comment:
This loading directive is working :smile: I found this when experimenting with Angular about a year ago, but can't remember where I found it.